### ISSUE TITLE
Feature request to implement quick access device control

### ISSUE THREAD

**Issue by SnoutBug** [Florianisme/WakeOnLan#3]
Its basically the same as the quick access you already implemented but more dynamic (from what i understand)

**Comment by Florianisme** [comment]
Sorry, what do you mean? I don't understand what's different to my existing Quick Tile implementation

**Comment by SnoutBug** [comment]
this is a rather [new feature introduced with android 11](https://developer.android.com/develop/ui/views/device-control)
this unites all device controls in one panel and is mainly used for home automation (kde connect uses them for commands, home assistant for automation)
the only difference would be that you would not have to hard code three quick tiles for three devices, but could generate them dynamically.
The user then has access to those via a button on the lock screen (if turned on in settings)
![Screenshot_20230106-195141](https://user-images.githubusercontent.com/71790678/211079320-7226023f-e0e8-493a-aa13-4a787f9f7b3f.png)
mentioned lockscreen button
![Screenshot_20230106-194946](https://user-images.githubusercontent.com/71790678/211078939-858bf107-e355-463d-abdf-9e639a9571c6.png)
example image of the device control feature as used by kde connect

**Comment by Florianisme** [comment]
Ah now I know what you mean!
Yes that looks interesting, I'll try to implement it!

**Comment by Florianisme** [comment]
The feature will land in beta in the next few hours. You can sign up here and leave me some feedback on it: https://play.google.com/store/apps/details?id=de.florianisme.wakeonlan

**Comment by SnoutBug** [comment]
Whoa that was fast. The feature looks good to me and works fine!
The device control does not seem to be controllable from the locked screen though (while home assistant is)

**Comment by Florianisme** [comment]
Thats strange, it's working for me from the locked screen both on my actual phone aswell as on the emulators.
Which device and OS are you using if I may ask?

**Comment by SnoutBug** [comment]
I'm using a Google Pixel 5 with GrapheneOS on Android 13. (TQ1A.230105.001.2023011000)
There might be some shenanigans on GOS side.

**Comment by Florianisme** [comment]
I'm going to try to run GrapheneOS on an emulator and see if I can replicate the issue. 

Do you know how to retrieve Logcat logs from your phone if my approach does not work?

**Comment by Florianisme** [comment]
I've rolled out the feature to the public Beta channel. Let's see if anyone uses it besides us :)

**Comment by danielwaltz** [comment]
Found and set up the app tonight! I can confirm that on the 1.6.5 beta the device control feature is working for me on a Pixel 7. Also the beta release indeed fixed the connection issues noted in the release notes that I was also experiencing when trying to set up waking from my Pixel Watch.

Just learned about wake on LAN tonight and I got it all set up for the first time. This app is already my favorite of the ones I tried, I love all the OS integration going on. It works great, looks great, is ad free, and has no tracking... it's fabulous. Just want to give credit where credit is due, you are the man and I appreciate your work! 🙏
